Runtime/Scripts/Export/StandardMaterialExport.cs

```diff
@@ -104,8 +104,17 @@ public override bool ConvertMaterial(UnityEngine.Material uMaterial, out Materia
                         skyboxData.exposure = uMaterial.GetFloat("_Exposure");
                         skyboxData.rotation = uMaterial.GetFloat("_Rotation") + 90f;
 #if UNITY_EDITOR
-                        uMaterial.mainTexture =
-                            ResizeTextureToImportSettings(uMaterial.GetTexture(k_Tex));
+                        UnityEngine.Texture texture = uMaterial.GetTexture(k_Tex);
+                        string texturePath = AssetDatabase.GetAssetPath(texture);
+                        if (Path.GetExtension(texturePath) == ".tif")
```

Comment:
Any way to get the mime types? because the extension could be `.tiff` or also in different case

Reply:
I had a quick look - can't find something in Unity, and the other APIs are in external C# DLLs. If it's just these two possibilities, I could check for either.

Reply:
yeah maybe lower the string and check for `tiff` and `tif` then 👍

Reply:
:ok_hand: Yeah, the libraries are in `System.Web` or `System.Drawing.Imaging`. Otherwise the alternative is to read the header ourselves, but it'll take a while to write that. I'll do the extension check.